src/System.Text.Json/src/System/Text/Json/Reader/JsonReaderOptions.cs

```diff
@@ -8,13 +8,35 @@ namespace System.Text.Json
     /// Provides the ability for the user to define custom behavior when reading JSON
     /// using the <see cref="Utf8JsonReader"/> that may deviate from strict adherence
     /// to the JSON specification (as per the JSON RFC - https://tools.ietf.org/html/rfc8259),
-    /// which is the default behavior.
+    /// which is the default behavior. It also lets the user define the maximum depth to read up to.
```

Comment:
It's not just about "using the Utf8JsonReader" anymore, right?  e.g. someone might be using JsonDocument, and it's an implementation detail that it's using Utf8JsonReader.

I would also just end the summary before the "that may deviate from...".  The additional text isn't all that meaningful, it's long (especially for popping up in IntelliSense), and we wouldn't want to have to update the summary in docs any time we add a property.